doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -7,27 +7,54 @@
+This chapter describes the steps to install Kubernetes based Reference Implementation (RI-2). The entire installation is divided into two stages - Host provisioning & Software provisioning. The host provisioning stage is provided here for information only and can be skipped when using Bare Metal Providers like Packet, etc.  The software provisioning stage is agnostic to the host provisioning stage, in that there is no dependency between the installer used for the software provisioning stage and any tools used in the host provisioning stage.
 
 <a name="4.2"></a>
 ## 4.2 Prerequisites
 
-> Describe the prerequisites needed in order to install the RI - e.g. all requirements in chapter03.md are met.
-
+Before attempting the installation, please ensure that all requirements described in [Chapter 3](./chapter03.md) are met.
+(More details to follow)
 
 <a name="4.3"></a>
 ## 4.3 Installation of the Reference Implementation
 
-> Describe the tasks required in order to install the RI.
+<a name="4.3.1"></a>
+### 4.3.1 Host Provisioning
+
+Host provisioning is the operation of preparing a host before the software stack can be installed on them. This includes (and not limited to) installing operating system, configuring network so that the hosts are reachable via SSH, configuring storage, etc.
+
+A former OPNFV baremetal provisioner XCI, now referred to as Cloud Infra Automation Framework and hosted by [Nordix Labs](https://docs.nordix.org/submodules/infra/engine/docs/user-guide.html#framework-user-guide) has been used in the host provisioning state. The mentioned framework uses [Bifrost](https://docs.openstack.org/bifrost/latest/) for provisioning virtual and baremetal hosts. It uses Ansible playbooks to automate the the deployment of a base image onto a set of known hardware using [Ironic](https://docs.openstack.org/ironic/latest/).
+
+Before initiating a deployment, two configuration templates, referred to as POD Descriptor File (PDF) and Installer Descriptor File (IDF) in OPNFV terminology need to be defined. Both PDF and IDF files are modelled as yaml schema.
+
+A PDF is a hardware configuration template that includes hardware characteristics of the jumphost host & the set of compute/controller hosts. For each host, the following characteristics should be defined:
+- CPU, disk & memory information
+- Remote management parameters
+- Network interfaces list including name, MAC address, IP address, link speed
+
+IDF includes infromation about network information required by the installer. All the networks along with possible VLAN, DNS and gateway information should be defined here.
+
+More details regarding these descriptor files as well as their schema is very well documented in [Chapter 7](https://github.com/cntt-n/CNTT/blob/master/doc/ref_impl/cntt-ri/chapters/chapter08.md#opnfv-descriptor-files-1) of RI-1.
```

Comment:
1. The text refers to Chapter 7 but the link is to Chapter 8. Since Chapter 7 is from RI-1, it may be better to list it as "RI-1 Chapter 7" rather than just as "Chapter 7" for clarity.
1. I think the recommendation is to use relative addresses (../../)

Reply:
Done